skbio/sequence/_rna.py

```diff
@@ -135,6 +135,73 @@ def degenerate_map(cls):
     def _motifs(self):
         return _motifs
 
+    @stable(as_of="0.4.0-dev")
+    def reverse_transcribe(self):
+        """Reverse transcribe RNA into DNA.
+
+        RNA sequence is assumed to be the coding strand. Uracil (U) is
```

Comment:
:+1: :heavy_multiplication_x: :100: for `sense` and `antisense` that is much more precise.